qemu/tests/pxe_query_cpus.py

```diff
@@ -15,7 +15,7 @@
 @error_context.context_aware
 def _capture_tftp(test, vm, timeout):
     error_context.context("Snoop packet in the tap device", logging.info)
-    output = aexpect.run_fg("tcpdump -nli %s" % vm.get_ifname(),
+    output = aexpect.run_fg("tcpdump -nli %s |grep -i tftp" % vm.get_ifname(),
```

Comment:
FYI: https://github.com/autotest/tp-qemu/pull/1931

And if you want to capture some messages, you can filter out `not arp and not icmp` or other.

Reply:
I have updated it, please help me review it again.Thanks.